src/indra_cogex/client/queries.py

```diff
@@ -870,15 +870,15 @@ def get_evidences_for_stmt_hashes(
 
 @autoclient()
 def get_stmts_for_paper(
-    term: Tuple[str, str], *, client: Neo4jClient, **kwargs
+    pmid_term: Tuple[str, str], *, client: Neo4jClient, **kwargs
```

Comment:
does this name make sense? you can give PMC, TRID, or DOI here too

Reply:
That update was needed to fix the api docs

Reply:
The dict that populates the api doc examples is keyed by the argument names

Reply:
But we can change `pmid_term` to `paper_id` or something similar

Reply:
Resolved in https://github.com/bgyori/indra_cogex/pull/134/commits/21ebf82b7d9a885e45f591581e5230719728b15a